tests/page-objects/reports/reports.wdio.page.js

```diff
@@ -19,8 +20,12 @@ const formTitle = () => $('#report-form #form-title');
 const submitButton = () => $('#report-form .form-footer .btn.submit');
 
 const forms = () => $$('.action-container .general-actions .actions.dropup .dropdown-menu li');
+const deselectReport = () => $('#reports-content .report-body .deselect');
+const itemSummary = () => $('#reports-content .report-body .item-summary');
```

Comment:
I think you can reuse `reportBody` here: 
```suggestion
const itemSummary = () => $(`${reportBody} .item-summary`);
```